cmd/meroxa/root/pipelines/update.go

```diff
@@ -44,9 +40,8 @@ type Update struct {
 	}
 
 	flags struct {
-		State    string `long:"state" usage:"new pipeline state (pause | resume | restart)"`
-		Name     string `long:"name" usage:"new pipeline name"`
-		Metadata string `long:"metadata" short:"m" usage:"new pipeline metadata"`
```

Comment:
Same as `pipelines create`, how come we're removing this?

Reply:
I forget which, it's either unused or has no use case for being exposed to customers. The only metadata example I can find in our official developer documentation is around creating a postgres resource.